Foundation/testsuite/src/ProcessRunnerTest.cpp

```diff
@@ -0,0 +1,332 @@
+//
+// ProcessRunnerTest.cpp
+//
+// Copyright (c) 2023, Applied Informatics Software Engineering GmbH.
+// Aleph ONE Software Engineering d.o.o.,
+// and Contributors.
+//
+// SPDX-License-Identifier:	BSL-1.0
+//
+
+
+#include "ProcessRunnerTest.h"
+#include "CppUnit/TestCaller.h"
+#include "CppUnit/TestSuite.h"
+#include "Poco/PIDFile.h"
+#include <sstream>
+#include "Poco/Util/PropertyFileConfiguration.h"
+#include "Poco/Format.h"
+#include "Poco/Path.h"
+#include "Poco/File.h"
+#include "Poco/FileStream.h"
+#include "Poco/Environment.h"
+#include <fstream>
+#include <iostream>
+
+
+//using namespace IoT::Devs::Util;
```

Comment:
remove

Reply:
Fixed in [b02c2cc](https://github.com/pocoproject/poco/pull/4225/commits/b02c2cc267557c0945d24709cddaced0e720e808)